### ISSUE TITLE
Launchpad: launchpad component

### ISSUE THREAD

**Issue by TimBroddin** [Automattic/wp-calypso#83850]
Create the component to display the Launchpad. This will involve three parts:

- The component
- The styling
- A hook

An example can be found in `client/my-sites/earn/hooks/launchpad `& `client/my-sites/earn/hooks/use-earn-launchpad-tasks.ts`

The hook can be implemented as an empty shell here. Interactivity will be added in another issue.

Add the component to the page behind a feature flag "subscribers-launchpad)

Depends on #83852
